Libraries/LibWeb/CSS/Parser/PropertyParsing.cpp

```diff
@@ -4438,4 +4442,25 @@ RefPtr<CSSStyleValue> Parser::parse_filter_value_list_value(TokenStream<Componen
     return FilterValueListStyleValue::create(move(filter_value_list));
 }
 
+RefPtr<CSSStyleValue> Parser::parse_view_transition_name_value(TokenStream<ComponentValue>& tokens)
+{
+    // none | <custom-ident>
+    tokens.discard_whitespace();
+    {
+        auto transaction = tokens.begin_transaction();
+
+        // The values 'none' and 'auto' are excluded from <custom-ident> here.
+        // Note: Only auto is excluded here since none isn't parsed differently.
+        auto ident = parse_custom_ident_value(tokens, { "auto"sv });
+        if (!ident)
+            return {};
+
+        tokens.discard_whitespace();
+        transaction.commit();
+
+        return CustomIdentStyleValue::create(ident->custom_ident().to_string());
```

Comment:
`ident` is already a `CustomIdentStyleValue`. :^)

Reply:
Changed.